grpc-bridge/src/main/java/org/jboss/resteasy/grpc/servlet/AsyncMockServletOutputStream.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.resteasy.grpc.servlet;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+public class AsyncMockServletOutputStream extends MockServletOutputStream {
+
+   private CountDownLatch latch = new CountDownLatch(1);
+
+   public void await() throws InterruptedException {
+      while (true) {
```

Comment:
Is there a reason for the loop? This would be endless if interrupted.

Reply:
Well, if the loop is interrupted, then it will go back to latch.await() and keep trying ...